parser-typechecker/src/Unison/PrintError.hs

```diff
@@ -1218,16 +1218,18 @@ rangeToEnglish (Range (L.Pos l c) (L.Pos l' c')) =
               then "line " ++ show l
               else "lines " ++ show l ++ "—" ++ show l'
 
-annotatedToEnglish :: (Annotated a, IsString s) => a -> s
+annotatedToEnglish :: (Annotated a, IsString s, Semigroup s) => a -> s
 annotatedToEnglish a = case ann a of
-  Intrinsic -> "an intrinsic"
-  External -> "an external"
+  Intrinsic -> "<intrinsic>"
+  External -> "<external>"
```

Comment:
Will this change make any existing sentences worse?

Reply:
I don't think I've ever seen these end up in error messages TBH, I think they're only used to annotate builtin terms which shouldn't have any compiler errors 🤞🏼